internal/ansible/controller/reconcile_test.go

```diff
@@ -342,42 +346,6 @@ func TestReconcile(t *testing.T) {
 				},
 			},
 		},
-		{
-			Name:            "reconcile deletetion",
-			GVK:             gvk,
-			ReconcilePeriod: 5 * time.Second,
-			Runner: &fake.Runner{
-				JobEvents: []eventapi.JobEvent{
-					eventapi.JobEvent{
-						Event:   eventapi.EventPlaybookOnStats,
-						Created: eventapi.EventTime{Time: eventTime},
-					},
-				},
-				Finalizer: "testing.io/finalizer",
-			},
-			Client: fakeclient.NewClientBuilder().WithObjects(&unstructured.Unstructured{
-				Object: map[string]interface{}{
-					"metadata": map[string]interface{}{
-						"name":      "reconcile",
-						"namespace": "default",
-						"annotations": map[string]interface{}{
-							controller.ReconcilePeriodAnnotation: "3s",
-						},
-						"deletionTimestamp": eventTime.Format(time.RFC3339),
-					},
-					"apiVersion": "operator-sdk/v1beta1",
-					"kind":       "Testing",
-					"spec":       map[string]interface{}{},
-				},
-			}).Build(),
-			Result: reconcile.Result{},
-			Request: reconcile.Request{
-				NamespacedName: types.NamespacedName{
-					Name:      "reconcile",
-					Namespace: "default",
-				},
-			},
-		},
```

Comment:
Why is this test now unnecessary?

Reply:
I bet this is because of the behaviour of the fake client which has changed in controller-runtime and now requires a sub resource to be present explicitly, which makes this unnecessary as the next test does the same. Should have commented while making this change :/

Let me run it locally and see if my brain remembers!